core/application/impl/app_configuration_impl.cpp

```diff
@@ -77,6 +80,21 @@ namespace {
     }
     return std::nullopt;
   }
+
+  std::optional<kagome::application::AppConfiguration::OffchainWorkerMode>
```

Comment:
Is there a reason why it's optional instead of result?

Reply:
It is temporary cell, for option which can be absent in args.